esp/esp/program/modules/migrations/0014_lotteryfrontendmodule.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```

Comment:
I don't think this should be here -- maybe it was missed elsewhere but I would have thought we'd have had it.